sync-core/src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -148,7 +148,6 @@ public void setRequestBody(InputStream input, long inputLength) {
         this.input = input;
         this.inputLength = inputLength;
     }
-
```

Comment:
Looks like you have a whitespace diff but this file is otherwise untouched - probably best to revert it?